<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Stop_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>520</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red</string>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="Video_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>641</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:b</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="masuk">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>92</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Masuk</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="keluar">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>530</y>
     <width>92</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Keluar</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>530</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="singleStep">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="harga">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>530</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Harga</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
